python/uyuni-cobbler-helper/uyuni-cobbler-helper.changes.cbosdo.profile-dl

```diff
@@ -0,0 +1 @@
+- Add helper function getting channels from a distribution tree
```

Comment:
see comment on main .changes. 

the  main .changes can't be empty or the  push to OBS will break. you can move what you put here in the main .changes when we will tag, tito will add the rest of the template for the changes.
the changelog check will fail but in this  specific case this is expected.